src/int.cr

```diff
@@ -514,6 +514,25 @@ struct Int
     yield ptr, count
   end
 
+  def inspect(io : IO) : Nil
```

Comment:
Why do you add this back? In #7525 it was decided to remove it.

Reply:
Probably a mistake.

Reply:
it was a merge conflict. this was a mistake.

Reply:
```sh
git reset --hard HEAD^
git fetch upstream
git rebase upstream/master
git push --force
```